No such file or directory - ci-scripts/create-release.js

### Description
On doing a fresh install of `npm install fundamental-react` for the latest version 0.3.0 I get an error for a missing file 'create-release.js':

> npm ERR! enoent ENOENT: no such file or directory, chmod 'projectdir/node_modules/fundamental-react/ci-scripts/create-release.js'
> npm ERR! enoent This is related to npm not being able to find a file.

If I do an install for v0.2.0 I don't get this error.

### Versions
**fundamental-react:** 0.3.0
**fiori-fundamentals:** 1.4.1

--- a/package.json
+++ b/package.json
@@ -9,9 +9,6 @@
     "type": "git",
     "url": "https://github.com/SAP/fundamental-react"
   },
-  "bin": {
-    "create-release": "ci-scripts/create-release.js"
-  },
   "scripts": {
     "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
     "build-doc": "node scripts/build.js",

